public/app/plugins/datasource/loki/queryHints.ts

```diff
@@ -26,6 +26,7 @@ export function getQueryHints(query: string, series: DataFrame[]): QueryHint[] {
           label: 'Consider using JSON parser.',
           action: {
             type: 'ADD_JSON_PARSER',
+            hint: 'hint: add json parser',
```

Comment:
Does the hint need to include `hint:` Wouldn't it be better without it, as now we are introducing naming convention, that might not be obvious to everyone and we might end up with some hints that have `hint:` and some that don't.

Reply:
Also, not sure if this would be part of this PR, but I remember we talked about renaming these hints to be more goal oriented. E.g. `extract fields using json parser`

Reply:
Sure, I just remember that we talked about the fact, that `add parser` doesn't tell user much and that we wanted to more focuse on goal of the hint. But I agree that it is out os scope for this PR.